src/Chapter1.hs

```diff
@@ -631,8 +626,14 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
+sumLast2 :: Int -> Int
+sumLast2 n = lastOne n + secToLast n
+  where
+    lastOne :: Int -> Int
+    lastOne n = mod (abs n) 10
+    secToLast :: Int -> Int
+    secToLast n = let nd = mod (div (abs n) 10) 10 in nd
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod (abs n) 10
div (abs n) 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂